src/main/java/net/william278/velocitab/Velocitab.java

```diff
@@ -144,12 +148,13 @@ public void log(@NotNull String message, @NotNull Throwable... exceptions) {
     }
 
     public PluginDescription getDescription() {
-        return server.getPluginManager().getPlugin("velocitab").get().getDescription();
+        return pluginContainer.getDescription();
     }
 
     private void registerCommands() {
+        final BrigadierCommand command = new VelocitabCommand(this).command();
         server.getCommandManager().register(
-                server.getCommandManager().metaBuilder("velocitab").build(),
-                new VelocitabCommand(this));
+                server.getCommandManager().metaBuilder(command).plugin(this).build(),
+                command);
```

Comment:
```suggestion
                command
        );
```
minor style nit -- could the closed parenthesis be moved to a newline with a method call with parameters on multiple lines?